Cubical/Experiments/CohomologyGroups.agda

```diff
@@ -1,140 +0,0 @@
-{-# OPTIONS --safe #-}
```

Comment:
@aljungstrom   Should this be kept or deleted?

Reply:
This one does not check.

Reply:
Commented out whole file.